src/asm2wasm.h

```diff
@@ -664,6 +664,10 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
     }, debug, false /* do not validate globally yet */);
   }
 
+  // if we see no function tables in the processing below, then the table still exists and has size 0
+
+  wasm.table.initial = wasm.table.max = 0;
```

Comment:
Turns out we already do that later down when we create the import - we mark the table as both imported and existing on consecutive lines. Seems ok that way to me.